lib/gds_api/publishing_api_v2.rb

```diff
@@ -95,7 +95,7 @@ def lookup_content_id(base_path:, exclude_document_types: nil, exclude_unpublish
   # @option options [String] locale The language, defaults to 'en' in publishing-api.
   #
   # @see https://github.com/alphagov/publishing-api/blob/master/doc/api.md#post-v2contentcontent_idpublish
-  def publish(content_id, update_type, options = {})
+  def publish(content_id, update_type = nil, options = {})
     params = {
       update_type: update_type
     }
```

Comment:
I agree, that would be better, but the problem is that apps currently call this method like `publish(content_id, "major")` and that change would break them all.